geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -190,34 +350,37 @@ These commands are supported for Redis 5.
 
    - uptime_in_days
 
- - stats
+- stats
 
-    - total_commands_processed
+   - total_commands_processed
 
-    - instantaneous_ops_per_sec
+   - instantaneous_ops_per_sec
 
-    - total_net_input_bytes
+   - total_net_input_bytes
 
-    - instantaneous_input_kbps
+   - instantaneous_input_kbps
 
-    - total_connections_received
+   - total_connections_received
 
-    - keyspace_hits
+   - keyspace_hits
 
-    - keyspace_misses
+   - keyspace_misses
 
-    - evicted_keys (always returns 0)
+   - evicted_keys (always returns 0)
 
-    - rejected_connections (always returns 0)
+   - rejected_connections (always returns 0)
 
-**[3]**  SLOWLOG is implemented as a NoOp.
+**[5]**  SLOWLOG is implemented as a NoOp.
 
 ## <a id="advantages-over-redis" class="no-quick-link"></a>Advantages of <%=vars.product_name%> over Redis
 
 <%=vars.product_name%>’s primary advantage is its **scalability**. While the Redis server is single threaded, <%=vars.product_name%> supports high concurrency. Many Redis clients can execute commands on the <%=vars.product_name%> cluster simultaneously.
 
 <%=vars.product_name%>'s architecture and management features help detect and resolve **network partitioning** problems without explicit management on the part of the Redis client.
 
+<%=vars.product_name%> for Redis partitions data across multiple servers and keeps replicated up to date _synchronously_, whereas Redis uses asynchronous replication.
+This provides a higher level of data consistency within the cluster.
+
 ## <a id="expiration-accuracy" class="no-quick-link"></a>Expiration Accuracy
 
 Keys are expired in two ways, actively and passively:
```

Comment:
The passive expiration description below is incorrect. Passive expiration is currently evaluated every 3 minutes.